src/components/actorScriptNav.jsx

```diff
@@ -13,11 +13,11 @@ const ActorScriptNav = ({
       <ActorScriptButton
         setCurrentActor={setCurrentActor}
         setCurrentCharacters={setCurrentCharacters}
-        currentActor={currentActor}
-        firstName={actor[0]}
-        lastName={actor[1]}
+        firstName={actor.firstName}
+        lastName={actor.lastName}
+        id={actor.id}
         scriptOption={scriptOption}
-        key={`${actor}Button`}
+        key={`${actor.firstName} ${actor.lastName}Button`}
```

Comment:
Not a suggestion, but I think another way this could be done would be like this:
```jsx
key='{actor.firstName} {actor.lastName}Button'
```

Reply:
Ah, but should we be using actorId here, rather than the name?

Reply:
The first suggestion doesn't work as far as I can tell. I think because it's jsx so if you want it to recognize anything js like it needs to be wrapped in curly brackets so it can't handle creating that string. Upcoming commit will be changing the key to be the actor id though.

Reply:
Interesting. Kind of disappointing that syntax doesn't work.